data/databases/create_dataset_from_mongodb_collection.py

```diff
@@ -0,0 +1,46 @@
+"""Create dataset from MonogDB"""
+
+# Author: Nicholas Png
+# Created: 31/01/2020
+# Last Updated: 20/02/2020
+
+import datatable as dt
+import pandas as pd
+from h2oaicore.data import CustomData
+
+
+_global_modules_needed_by_name = ["pymongo", "dnspython"]
```

Comment:
Why do you need `_global_modules_needed_by_name` and then also `_modules_needed_by_name`?

Reply:
true... I think I only need the first. But `pymongo` is not installed without `_global_modules_needed_by_name`